client/src/handlebars/helpers.side-effects.js

```diff
@@ -384,7 +384,13 @@ function subject_name(subject){
     );
 
   } else {
-    return subject.name;
+    if (subject.name === "Government"){
+      return "the Government";
```

Comment:
Just out of curiosity, why was this needed?

Reply:
Fixing a grammar mistake - since it was set to show the subject name, the text will read "Government ..." when in fact the grammatically correct way of saying it would be "The Government ..." 

Same with the version in french

@ktw1016